package.json

```diff
@@ -543,6 +543,14 @@
                 "key": "f4",
                 "command": "java.action.showTypeHierarchy",
                 "when": "editorLangId == java && editorTextFocus"
+            },
+            {
+                "mac": "shift+cmd+up",
```

Comment:
shift+cmd+up is not the right keybinding for macOS. But no keybinding seem to work on Eclipse for macOS. So I would leave the binding to alt+shift+up here as well.